packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```
@@ -89,10 +89,12 @@
         justify-content: space-between;
         margin: 2.4em 0;
         padding: 1.6rem;
+        width: calc(100% - 24px);
```

Suggestion to use `2.4rem` instead of `24px` since px is not scalable.

```suggestion
        width: calc(100% - 2.4rem);
```